packages/shared/src/variants/__tests__/quantum.test.ts

```diff
@@ -182,6 +182,8 @@ test("Two passes ends the game", () => {
 
   game.playMove(0, "pass");
   game.playMove(1, "pass");
+  
+  expect(game.getSGF()).toBe("(;\nEV[GO Variants]\nPB[player Black]\nPW[player white]\nSZ[4:2]\nKM[6.5]\nVS[quantum]\n\n\n;B[ba];W[ca];B[bb];W[cb]\n\n)");
```

Comment:
Ah I think I understand your comment on the other PR now

> Are there any benefits to creating a more complex test?

I had written a standalone test, which has some benefits, but this is fine too :)

Reply:
Perfect!